doc/DOCUMENTATION.org

```diff
@@ -421,7 +421,7 @@ contribution guidelines in [[./CONTRIBUTE.org][CONTRIBUTE.org]].
 
 ** Example: Themes Megapack example
 This is a simple =contrib= configuration layer listing a bunch of themes, you
```

Comment:
I think so because it refers to the concept of layers contributed by other people and not the directory here.